### ISSUE TITLE
Release 5.0.3

### ISSUE THREAD

**Issue by mrienstra** [A11yance/aria-query#430]
The last release ([v5.0.2](https://github.com/A11yance/aria-query/releases/tag/v5.0.2)), did not contain various version bumps made on Aug 25th, in particular, https://github.com/A11yance/aria-query/pull/395 bumped several `@babel` packages, possible enough to resolve this warning:
```sh
core-js-pure@npm:3.21.1 is deprecated: core-js-pure@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js-pure.
```
(seen while install dependencies for https://github.com/algolia/docsearch locally)

Thanks!

**Comment by jessebeach** [comment]
@mrienstra thank you for reporting this. I'm going to bump to 5.1.0 tomorrow with several other changes.

**Comment by jessebeach** [comment]
@mrienstra v5.1.1 is now available. That should eliminate the issue you're seeing.

**Comment by mrienstra** [comment]
Thank you @jessebeach, I'll spread the word!

**Comment by mrienstra** [comment]
Whoops, didn't realize this only affects `core-js@<3.23.3` & `core-js-pure@<3.23.3`, but not `core-js-compat@<3.23.3` -- for `core-js-compat` it's `core-js-compat@<3.2.0`. 🤦

**Comment by jessebeach** [comment]
@mrienstra I don't understand the implication. What do you think needs to happen?

**Comment by mrienstra** [comment]
Sorry @jessebeach, my bad, lemmie try again:

1. When I opened this issue, I mentioned a deprecation warning (`core-js-pure@<3.23.3 is no longer maintained and not recommended for usage`).
2. I was under the impression that this error messages was shown with (and thus applied to) `core-js`, `core-js-pure`, & `core-js-compat` (whenever version is less than `3.23.3`).
3. After taking a closer look, I realized the deprecation warning is shown for:
    - `core-js@<3.23.3`
    - `core-js-pure@<3.23.3`
    - `core-js-compat@<3.2.0` <-- different than what I originally thought.
4. I thought I should mention that here, just to reduce the chance that my initial misunderstanding will confuse others.
5. Everything you did was great, thanks for releasing a new version! No further action is needed on your end.
